grpc_opa/authorizer.go

```diff
@@ -240,7 +240,10 @@ func (a *DefaultAuthorizer) Evaluate(ctx context.Context, fullMethod string, opa
 			trace.StringAttribute("out", string(raw)),
 		}, "out")
 	}
-
+	// adding obligations data to context if present
+	if ob,ok := response["obligations"].([]string);ok{
```

Comment:
can you run this through go fmt? Apparently we don't check for that on build right now.